bom/openhab-addons/pom.xml

```diff
@@ -1,965 +1,970 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please check why this file (and CODEOWNERS) show up as fully changed.

Reply:
fixed, was a problem with the line endings